specifyweb/frontend/js_src/lib/components/InitialContext/treeRanks.ts

```diff
@@ -97,7 +97,18 @@ export const treeRanksPromise = Promise.all([
           )
       : []
   )
+  /*
```

Comment:
@acwhite211

Reply:
I changed discipline.taxontreedef from a one-to-one to many-to-one relationship, no django migration required for that, in this commit https://github.com/specify/specify7/pull/5032/commits/b7f9afb4c9ddcf63bb4471f3ba9e629db643c1b1